rasa_core/policies/ensemble.py

```diff
@@ -194,6 +196,30 @@ def load(cls, path):
         ensemble = ensemble_cls(policies, fingerprints)
         return ensemble
 
+    @classmethod
+    def load_from_yaml(cls, path):
+        # type: (Text) -> List[Policy]
+
+        config_data = utils.read_yaml_file(path)
+
+        policies = []
+
+        for policy in config_data.get('policies'):
+
+            module_name, _ , class_name = policy.get('name').rpartition('.')
```

Comment:
We can use the utils function if the policy path is fully specified. If the policy name is `KerasPolicy` we would need to import `KerasPolicy' in utils so that the name `KerasPolicy` is defined in `globals()`. This would lead to a circular import as `Policy` imports `utils`.

Reply:
I think all the policies are imported at package level so what should work is, within this function (not at the top of the file, but within the functions scope) do from rasa_core.policies import *